src/main/scala/ai/privado/languageEngine/javascript/tagger/sink/JSAPITagger.scala

```diff
@@ -50,5 +50,41 @@ class JSAPITagger(cpg: Cpg, ruleCache: RuleCache) extends APITagger(cpg, ruleCac
       }
       storeForTag(builder, scriptTag, ruleCache)(Constants.apiUrl + newRuleIdToUse, domain._1)
     })
+
+    // Identification of script tag generated dynamically in code
+    // const n = document.createElement("script");
+    // n.type = "text/javascript";
+    // n.async = !0;
+    // n.src = "https://cdn.segment.com/analytics.js/v1/" + t + "/analytics.min.js";
+    // Tag the respective templateDom node as API sink
+    val parentBlocksOfHTMLScriptElement = cpg
+      .call("<operator>.fieldAccess")
```

Comment:
There is a constant defined for this one within Joern library `io.shiftleft.codepropertygraph.generated.Operators.fieldAccess`